CONTRIBUTING.rst

```diff
@@ -170,7 +196,7 @@ but will cause spurious test failures.
 
     See ``doc/README.txt`` for details on working with the documentation.
 
--   (Optional) Activate GitHub and Appveyor for your forked GitHub repository,
+-   (Optional) Activate GitHub for your forked GitHub repository,
     so that the cross-platform compatibility tests get run
     whenever you push your work to your repository.
     These tests will also be run
```

Comment:
This item can be deleted, or replaced by a mention that the tests WILL be run automatically when using GitHub, no activation needed. Well, this might require #471 to be merged first :)

Reply:
Now I can remove it. :-)

Reply:
I'll remove also the AppVeyor CI batch and all other references to AppVeyor from doc.